gui/pages/hostname.go

```diff
@@ -0,0 +1,137 @@
+// Copyright © 2018 Intel Corporation
+//
+// SPDX-License-Identifier: GPL-3.0-only
+
+package pages
+
+import (
+	"github.com/clearlinux/clr-installer/gui/utils"
+	"github.com/clearlinux/clr-installer/hostname"
+	"github.com/clearlinux/clr-installer/model"
+	"github.com/gotk3/gotk3/gtk"
```

Comment:
Please move the 3rd party gtk package above our packages separated by a blank line.

Our import policy has been: 
- Core golang modules (no path0
- 3rd party (include path)
- local code